cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -69,15 +69,15 @@ endif
 
 GPI_EXTRA:=
 ifeq ($(TOPLEVEL_LANG),vhdl)
-    VSIM_ARGS += -foreign \"cocotb_init $(LIB_DIR)/libfli.$(LIB_EXT)\"
+    VSIM_ARGS += -foreign \"cocotb_init $(LIB_DIR)/libfli_modelsim.$(LIB_EXT)\"
```

Comment:
```suggestion
    VSIM_ARGS += -foreign \"cocotb_init $(LIB_DIR)/libcocotbfli_modelsim.$(LIB_EXT)\"
```

Reply:
Probably need to also change here: https://github.com/cocotb/cocotb/blob/master/cocotb/share/lib/fli/FliImpl.cpp#L1050
and here https://github.com/cocotb/cocotb/blob/master/cocotb/share/lib/fli/entrypoint.vhdl#L12. I Will check.

Reply:
Is `entrypoint.vhdl` needed?

Reply:
I have no idea.

I did notice that `libfli` seems to already be a library name used by other software, so getting `cocotb` in the filename seems important

Reply:
It looks to me `entrypoint.vhdl` was some intermediate when developing FLI and the library is loaded now by  `VSIM_ARGS`. I will check. If not used make a new PR?

>I did notice that libfli seems to already be a library name used by other software, so getting cocotb in the filename seems important

OK.

Reply:
> If not used make a new PR?

Sounds good to me